gradle-plugin/src/main/groovy/io/realm/gradle/RealmRobolectric.groovy

```diff
@@ -0,0 +1,35 @@
+package io.realm.gradle
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class RealmRobolectric implements Plugin<Project> {
+
+    @Override
+    void apply(Project project) {
+        project.task('downloadRealmRobolectric') {
+            doLast {
+                def targetZip = "${project.projectDir}/realm-robolectric-darwin-${Version.VERSION}.zip"
+                def targetLib = "${project.projectDir}/robolectricLibs/librealm-jni-darwin.dylib"
+
+                if (project.file(targetZip).exists() && project.file(targetLib).exists()) {
+                    println "librealm-jni-darwin.dylib is already exsit. Skip download the Realm Robolectric."
+                    return
+                }
+
+                project.download {
+                    src "http://static.realm.io/downloads/java/realm-robolectric-darwin-${Version.VERSION}.zip"
+                    dest "${project.projectDir}"
+                }
+
+                project.copy {
+                    from project.zipTree(project.file(targetZip))
+                    into "${project.projectDir}/robolectricLibs"
```

Comment:
I hope that `realm-robolectric-darwin.dylib` does not influence app's APK package. If there is something in `lib`, it would be included in APK?

If we store`realm-robolectric-darwin.dylib` in `lib` folder, we should modify `java.library.path` too.